src/select/searchable.rs

```diff
@@ -511,8 +538,6 @@ where
                         />
                     </div>
                     <div class="scrollable-table-wrapper" style={style_scrollable_table}>
-                    <div class="searchable-select-list-search-space" style={style_inner_width.clone()}>
-                    </div>
```

Comment:
I believe this part should not be removed as it may impact other design systems beyond Pumpkin. Let me share screenshots in the UI repository for further clarification.

Reply:
I have pushed the latest commit with adjustments to resolve style inconsistencies in other features.